source/includes/_files.md

```diff
@@ -0,0 +1,208 @@
+    "properties": {
+        "mimetype": "text/csv"
+    },
+    "created": "2017-06-07T15:16:05.701266+00:00"
+}
+```
+
+The file object contains:
+
+* a unique identifier
+* a URL for use in the platform
+* the file name
+* a set of file properties
+* a creation date
+
+### Attributes
+
+Attribute | Description
+--------- | -----------
+`file_id` <br> *string* | Unique identifier for the file
+`url` <br> *string* | Generated internal URL, using the `odsfile` scheme
+`filename` <br> *string* | Human readable name of the file
+`properties` <br> *object* | Set of file properties, such as its mimetype
+`created` <br> *string* | Time at which the file was created
+
+## List all files
+
+This endpoint lists all files available to the user who performs the request. A file is available to the requesting user if either the user uploaded the file themselves, or if the user has sufficient permissions to edit a dataset where the file is used, or if the user is a domain administrator who has sufficient permissions to edit all datasets of the domain.
+
+> Definition
+
+```HTTP
+GET https://{DOMAIN_ID}.opendatasoft.com/api/management/v2/files/
+```
+
+> Example response
+
+```json
+[
+    {...},
+    {
+        "file_id": "cheese_data.csv",
+        "url": "odsfile://cheese_data.csv",
+        "filename": "Cheese Data.csv",
+        "properties": {
+            "mimetype": "text/csv"
+        },
+        "created": "2017-06-07T15:16:05.701266+00:00"
+    },
+    {...}
+]
+```
+
+## List one file
+
+This endpoint is for retrieving the file object with provided file_id.
+
+> Definition
+
+```HTTP
+GET https://{DOMAIN_ID}.opendatasoft.com/api/management/v2/files/{FILE_ID}/
+```
+
+> Example request
+
+```shell
+curl 'https://yourdomain.opendatasoft.com/api/management/v2/files/cheese_data.csv'
+```
+
+> Example response
+
+```json
+{
+    "file_id": "cheese_data.csv",
+    "url": "odsfile://cheese_data.csv",
+    "filename": "Cheese Data.csv",
+    "properties": {
+        "mimetype": "text/csv"
+    },
+    "created": "2017-06-07T15:16:05.701266+00:00"
+}
+```
+
+
+## Upload a file
+
+This endpoint is for uploading a new file to the platform.
+There are two ways to send files to the platform: either by sending a full file using the HTTP multipart mechanism, or by sending the file content and other metadata such as its type and file name. The use case dictates which method works best. For instance, in order to create files out of batch processes it might be easier to use the content method, but to upload an existing file from a local hard drive, the multipart method might work better. Both methods have the same result: they create a file on the domain.
+
+> Definition
+
+```HTTP
+POST https://{DOMAIN_ID}.opendatasoft.com/api/management/v2/files/
+```
+
+This endpoint can receive a file sent in multipart, or file content sent through the content POST parameter, along with a mimetype and an optional filename.
+
+### Multipart file upload
```

Comment:
I think that we want to clarify the 240 Mo limit.

Reply:
:+1: makes sense

Reply:
(done in bb3a102)